packages/flame_test/lib/src/flame_test.dart

```diff
@@ -116,10 +120,26 @@ class GameTester<T extends Game> {
   ///
   /// Use [verify] closure to make verifications/assertions.
   @isTest
+  @Deprecated('Use testGameWidget instead')
   void widgetTest(
     String description,
     WidgetVerifyFunction<T>? verify,
   ) {
+    testGameWidget(description, verify: verify);
+  }
+
+  /// Creates a [Game] specific test case with given [description]
+  /// which runs inside the Flutter test environment.
+  ///
+  /// Use [setUp] closure to prepare your game instance (e.g. add components to
+  /// it)
+  /// Use [verify] closure to make verifications/assertions.
+  @isTest
+  void testGameWidget(
+    String description, {
+    WidgetSetupFunction<T>? setUp,
+    WidgetVerifyFunction<T>? verify,
```

Comment:
The verify should be required right?

Reply:
We didn't required it on the previous API, so I kept it the same way, not sure if we should require on this new API but usually I don't testing libraries to require this, I think `bloc_test` also don't require it.